src/components/select/select.bemhtml

```diff
@@ -0,0 +1,17 @@
+block('select')(
+    content()(function() {
+        const React = require('react');
+        const Popup = require('../src/components/popup/popup');
+
+        return [
+            { elem : 'button' },
+            <Popup
+                directions={['bottom-left', 'bottom-right', 'top-left', 'top-right']}
+                target='anchor',
+                theme={this.mods.theme}
+                autoclosable={true}>
+                {{ block : this.block, mods : this.mods, elem : 'menu' }}
```

Comment:
@nik-kor I found very very big trouble! Any ideas what we must do if we need react component in composition and it must contain bemjson? We can check this in components and use provide for render but I'm not sure.